### PR TITLE
WSO-270 - Schedule Brick does not scroll properly

### PR DESCRIPTION
**Pull request by TDRoden**

### CODE REVIEW

**Comment by SandyVallee** [comment]
per review complete

File: sites/all/themes/ucgy_theme/resources/js/owl-carousel/owl.carousel.js [hunk 1]
```diff
@@ -584,7 +584,8 @@ if (typeof Object.create !== "function") {
             if (base.options.scrollPerPage === true && base.currentItem > 0 && base.currentItem < base.options.items) {
                 base.currentItem = 0;
             } else {
-                base.currentItem -= base.options.scrollPerPage === true ? base.options.items : 1;
+                var shiftItemTotal = (base.currentItem % base.options.items) ? (base.currentItem % base.options.items) : base.options.items;
+                base.currentItem -= base.options.scrollPerPage === true ? shiftItemTotal : 1;
```
Comment (taeram) [inline sites/all/themes/ucgy_theme/resources/js/owl-carousel/owl.carousel.js]: Could you expand these two lines into two individual if/else statements? The ternary if statement is often a great way to shorten the code, but in this particular case, I think it just makes things a lot more confusing.

**Comment by TDRoden** [comment]
Test comment.

File: sites/all/themes/ucgy_theme/resources/js/brick_styling.js [hunk 2]
```diff
@@ -31,13 +31,16 @@
...
+            else if ($(bricks[i]).hasClass('rounded-top')) {
                 addBrickClass(bricks, 'rounded-brick-below', i, -1);
             }
             else if ($(bricks[i]).hasClass('rounded-bottom')) {
                 addBrickClass(bricks, 'rounded-brick-above', i, 1);
             }
-            else if ($(bricks[i]).hasClass('overlay')) {
+
+            // Timeline/Schedule brick with 'overlay' is also 'rounded'
+            if ($(bricks[i]).hasClass('overlay')) {
```
Comment (TDRoden) [inline sites/all/themes/ucgy_theme/resources/js/brick_styling.js]: This was a 'doh' moment!

Comment (TDRoden) [inline sites/all/themes/ucgy_theme/resources/js/brick_styling.js]: And I can leave comments now!

**Pull request merged**